js/telemetry/trackingEvents.ts

```diff
@@ -1,12 +1,19 @@
 import { SideBarSelection } from "../components/ladderPage/sidebar";
 import { estimatedArrival } from "../models/tripUpdate";
-import { FullStory } from "@fullstory/browser";
+import {
+  FullStory,
+  isInitialized as isFullStoryInitialized,
+} from "@fullstory/browser";
 
 export enum FullStoryEventName {
   SideBarOpened = "Orbit: Ladder Side Bar Opened",
 }
 
 export const trackSideBarOpened = (selection: SideBarSelection) => {
+  if (!isFullStoryInitialized()) {
+    return;
+  }
+
```

Comment:
nit: would it be better for us to wrap `FullStory()` so further tracking functions in this file don't need to repeat this?

Reply:
@mathcolo I think I follow? Took a stab at that in 5d72dda. See what you think. My only criticism with it is that now the individual event functions may be doing extra unnecessary work leading up to the FS call, rather than doing the init check first as an early-out.